src/Action/ResetAction.php

```diff
@@ -105,7 +105,7 @@ public function __construct(
         UserManagerInterface $userManager,
         LoginManagerInterface $loginManager,
         TranslatorInterface $translator,
-        SessionInterface $session,
+        Session $session,
```

Comment:
Could you please take a look at https://github.com/sonata-project/SonataUserBundle/pull/1184#issuecomment-647620411